site/docsource/Installation.adoc

```diff
@@ -16,7 +16,7 @@ is installed, run the following command:
 
 [source,sh]
 ------------------------------
-npm install --save bs-platform
+npm install --save-dev bs-platform
```

Comment:
@joshaber so users have to install typings separately to the library even though the library is written in typescript itself?